### ISSUE TITLE
WCAG 2.2 Violation - There are instances of user-initiated changes to content not being announced to the user.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#19]
### Issue

There are instances of where the fact that user-initiated changes to content have successfully occurred is not being announced to the user (and where neither of the following alternatives are true):
1. The page has undergone a full reload.
2. The changed content appears _after_ the control(s )that create the changes in content in the page's reading order.
3. There is a text description of the expected behaviour included _before_ the control(s) that create the changes in content in the page's reading order.

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [4.1.3 Status Messages](https://www.w3.org/TR/WCAG22/#status-messages)

### User Impact

Some users - such as screen reader users and braille display users - will be unaware that changes in content have successfully occurred.

#### Further information:

1. [The "Intent" section of "Understanding Success Criterion 4.1.3: Status Messages | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html#intent)

### Instances

1. #53 

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader.
2. Navigate through the page's content, noticing that there is no text description of the expected behaviour included _before_ the control(s) that create the changes in content in the page's reading order.
3. Navigate to the control(s) that create the changes in content, and initiate a change in content.
4. Notice that the page doesn't undergo a full reload.
5. Notice that the change in content is not announced to the screen reaer.
6. Notice that the changed content doesn't follow the controls within the page's reading order.

### How To Fix The Issue

Implement one or more of the following solutions:

1. Ensure that when the content is updated, the fact that the content has been updated is announced via live region (example: "Content updated.").
2. Include a text description of the expected behaviour _before_ the controls.
3. (Only if actually tenable, given the context) Place the changed content _after_ the controls in the page's reading order.

#### WCAG 2.2 Conformant Code Example

Not available/not applicable.
